client/landing/stepper/declarative-flow/flows/site-migration-flow/site-migration-flow.ts

```diff
@@ -127,11 +140,7 @@ const siteMigration: Flow = {
 			triggerGuidesForStep( flowName, currentStep, siteId );
 		}, [ flowName, currentStep, siteId ] );
 
-		// TODO - We may need to add `...params: string[]` back once we start adding more steps.
```

Comment:
Good catch. `params` have been removed from the signature, so this is a very stale TODO!